src/common/AddItemForm/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, memo } from 'react'
+import { TextField } from '@mui/material'
+
+import { useFormik } from 'formik'
+
+import styled from 'styled-components'
+
+type AddItemFormPropsType = {
+  callBack: (title: string) => void
+  label?: string
+}
+
+type ErrorsType = {
+  title: string
+}
+
+export const AddItemForm: FC<AddItemFormPropsType> = memo(({ callBack, label }) => {
+  const formik = useFormik({
+    initialValues: {
+      title: '',
+    },
+    validate: values => {
+      const errors = {} as ErrorsType
+      if (!values.title) {
+        errors.title = 'Title is required'
+      }
+      return errors
+    },
+    onSubmit: values => {
+      callBack(values.title)
+      formik.resetForm()
+    },
+  })
+
+  const { onChange, value, name } = formik.getFieldProps('title')
```

Comment:
instead of it, better to use Formik Field component https://formik.org/docs/api/field